packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -164,13 +162,11 @@ describe("No Delta Stream", () => {
 
         assert.strictEqual(container.connected, true, "container.connected");
         assert.strictEqual(container.clientId, "storage-only client", "container.clientId");
-        assert.strictEqual(container.readonly, true, "container.readonly");
-        assert.strictEqual(container.readonlyPermissions, true, "container.readonlyPermissions");
```

Comment:
Since it was in storage-only mode, I was unable to replicate the test for `readonlyPermissions` using the `readOnlyInfo` property